docs/PacketFence_MSPKI_Quick_Install_Guide.asciidoc

```diff
@@ -0,0 +1,360 @@
+
+Assumptions
+-----------
+
+* You have at least one server with PacketFence 5.4 or later.
+* The server already has a properly configured switch with 802.1X support.
+* The PacketFence RADIUS server is working in your environment.
+* You have a Microsoft Windows 2008 R2 Entreprise server installed.
+* The PacketFence management IP will be 192.168.1.5.
+* The RADIUS shared secret is "useStrongerSecret".
+* In this guide you will see a lot of use of <ServerDNSName>, most of the MSPKI services requires in their configuration to use the FQDN of the server and not his IP.
+
+Installation
+------------
+
+Step 1: Install Active Directory Certificate Service (ADCS)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+NOTE: This section will cover the configuration for Active Directory Certificate Services (ADCS) on Microsoft Windows 2008 R2 Entreprise. The installation of ADCS is not covered by this guide, refer to the Microsoft white paper about it for more information (http://social.technet.microsoft.com/wiki/contents/articles/9063.network-device-enrollment-service-ndes-in-active-directory-certificate-services-ad-cs.aspx).
+
+For the integration with PacketFence, the following subroles need to be installed in ADCS:
+
+* Certification Authority Web Enrollment
+* Network Device Enrollment Service
+* Online Responder
+
+Before you start the configuration, a hotfix is necessary due to a Microsoft issue. After restarting the ADCS service, the server cannot enroll new certificates and display the following error message: "The RPC Server is unavailable". The hotfix is available here: https://support.microsoft.com/en-us/kb/2633200
+
+Communication between the MSPKI and PacketFence will be using port 80.
+
+Configuring Network Device Enrollment Service (NDES)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For the deployment of ADCS you will need to configure Network Enrollment Device Service (NDES). This subrole will allow us to exchange certificates with the MSPKI server via Simple Certificate Exchange Protocol (SCEP).
+
+Every configuration change has to be done by an account with administrative privileges.
+
+Challenge Password
+^^^^^^^^^^^^^^^^^^
+
+Microsoft SCEP (MSCEP) includes by default a challenge password, which is unique and dynamically generated for each device which wants to enroll. In a BYOD deployment, this can be a barrier as a user cannot register a device by himself without the intervention of an administrator. Since we use NDES with PacketFence, our security to obtain a certificate would be the credentials necessary to access the enrollment system.
+
+To disable the challenge password you need to modifiy the following key in the Windows registry. 
+
+Click `Start` and enter `regedit`.
+
+Navigate to `Computer -> HKEY_LOCAL_MACHINE -> SOFTWARE -> Microsoft -> Cryptography -> MSCEP -> EnforcePassword`.
```

Comment:
This is still not good

The syntax for paths is : 
Computer->HKEY_LOCAL_MACHINE->SOFTWARE->Microsoft->Cryptography->MSCEP->EnforcePassword

No ticks and no spaces between the arrows

Change this in all the document